components/editor/prosemirror-schema/create-new-schema.md

```diff
@@ -0,0 +1,285 @@
+---
+title: Create a New Schema
+page_title:  Create a New Schema
+description: Expore how to create a new schema for the Editor for Blazor.
+slug: editor-create-new-schema
+tags: telerik,blazor,editor,prosemirror,new,schema
+published: True
+position: 5
+---
+
+# Create a New Schema
+
+This article describes how you can create a new [ProseMirror schema]({%slug editor-prosemirror-schema-overview%}) for the Editor to use. Creating a new schema is useful if you want to change the majority of the default schema.
+
+@[template](/_contentTemplates/editor/general.md#prosemirror-schema-prerequisites)
+
+## Basics
```

Comment:
Similar to the modify article, this section is not being rendered (neither the heading, nor the template-based contents).